ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/params/ParametersPanel.java

```diff
@@ -0,0 +1,51 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.console.params;
+
+import com.intellij.codeInsight.hint.HintManager;
+import com.intellij.json.JsonFileType;
+import com.intellij.openapi.components.ServiceManager;
+import com.intellij.openapi.editor.EditorFactory;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.EditorTextField;
+import com.intellij.util.messages.MessageBus;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.console.GraphConsoleView;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.console.event.QueryParametersRetrievalErrorEvent;
+import org.codehaus.jackson.JsonParseException;
+
+import java.awt.*;
+
+import static com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.console.event.QueryParametersRetrievalErrorEvent.*;
+
+public class ParametersPanel implements ParametersProvider {
+
+    private EditorTextField editor;
+
+    public void initialize(GraphConsoleView graphConsoleView, Project project) {
+        MessageBus messageBus = project.getMessageBus();
+        editor = new EditorTextField(EditorFactory.getInstance().createDocument("// Provide query parameters in JSON format here:"),
```

Comment:
Use scratch files here. Document created in such way will be deleted on IDE restart.
Scratch file will be persisted.

I have code samples somewhere in project, search for "Scratch"